jquery/Script/StudentManagerJquery.js

```diff
@@ -35,28 +31,22 @@ function StudentListGeneric() {
             function successd() {
                 StudentListGeneric();
             }
-            // deleteStudent(id);
         });
         $('.editBtn').click(function () {
-            debugger;
             var id = parseInt($(this).attr('data-id'));
             var url = baseUrl + 'Student?id=' + id;
             var parent = $(this).closest('tr');
-
-            // var student=ajaxCall(url,"get");
             serverCall(url, "get", null, successGetStudent);
-
-            function successGetStudent(student) {
-                var row = "<td>  <input id='Name' type='text'  value='" + student.Name + "'/></td>";
-                row += "<td>  <input id='FName' type='text'  value='" + student.FName + "'/></td>";
-                row += "<td>  <input id='email' type='text'  value='" + student.Email + "'/></td>";
-                row += "<td>  <input id='phone' type='text'  value='" + student.Phone + "'/></td>";
-                row += "<td>  <input id='DOB' type='date' value='" + student.Dob + "'/></td>";
-                row += "<td>  <input id='Password' type='password' value='" + student.Password + "'/></td>";
-
+            function successGetStudent(data) {
+                var row = "<td>  <input id='Name' type='text'  value='" + data.student.Name + "'/></td>";
```

Comment:
Could have used template literals instead of adding strings, would have saved time and is more understandable
E.g `<td value="${ student.Email }"></td>` etc.